wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests/Angular2CalculatorTest.wiki

```diff
@@ -0,0 +1,28 @@
+Sample test on [[Angular][https://angular.io/]] application. 
+
+!note '''Please note:''' The application used in this test does NOT seem to work with !-PhantomJs-!. It does work when executed with Chrome or Firefox.
+
+!note Due to the nature of [[Plunker][https://plnkr.co/]] which hosts the application in an iframe, the page must first be opened with !-BrowserTest-!, switch to the embedded iframe and wait for the application to load.
```

Comment:
It's needed when there is something async happening which then updates the page (usually http request, in the calculator it's just a setTimeout), then the "wait for angular" call waits for anything async to be completed. Try the calculator test with BrowserTest and it will fail,  If you want to stick with the calculator I can build a compiled version and upload to github pages?